.github/scripts/synchronize_jira_xray.js

```diff
@@ -205,6 +205,101 @@ async function updateJiraIssues(testSelfs, targetVersions, componentsList) {
       }
 
       core.info(`Issue ${api} updated successfully in Jira.`);
+
+      // Update status of the test
+      const statusAPI = `https://centreon.atlassian.net/rest/api/2/issue/${existingIssueData.key}/transitions?expand=transitions.fields`;
+
+      const issueStatusPayloadToReadyForImplementation = {
+        transition: { id: "61" },
+      };
+      const jiraStatusToReadyForImplementationResponse = await axios.post(
+        statusAPI,
+        issueStatusPayloadToReadyForImplementation,
+        {
+          headers: {
+            Accept: "application/json",
+          },
+          auth: {
+            username: JIRA_USER,
+            password: JIRA_TOKEN_TEST,
+          },
+        }
+      );
```

Comment:
can you create a function `postIssueStatus` in order to reuse it with `issueStatusPayloadToReadyForImplementation` , `issueStatusPayloadToStart` , `issueStatusPayloadToResolved`